README.md

```diff
@@ -34,13 +55,11 @@ When running on a non-linux system, the following environment variables are requ
 - `SPLUNK_MEMORY_LIMIT_MIB` (no default): Maximum total memory to be allocated by the process heap.
 - `SPLUNK_MEMORY_SPIKE_MIB` (no default): Maximum spike between the measurements of memory usage.
 
-Deploy the collector as outlined in the below. More information
-about deploying and configuring the collector can be found
-[here](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html)
+Deploy the collector as outlined in the below.
 
 ### Docker
 
-Deploy from a Docker container (replace `0.1.0` with the latest stable version number if necessary):
+Deploy from a Docker container (replace `0.1.0` with the latest stable version number):
```

Comment:
```suggestion
Deploy from a Docker container. Replace `0.1.0` with the latest stable version number:
```

is there a reason we specify 0.1.0 here? can we replace `0.1.0` with `<version>` and tell users to replace it with the latest stable version instead?

Reply:
Could do it though one is a valid config and other is not. Currently like this as it follows product docs: https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html.